code/game/dna/dna_modifier.dm

```diff
@@ -255,6 +242,18 @@
 		return
 	return ..()
 
+/obj/machinery/dna_scannernew/crowbar_act(mob/user, obj/item/I)
+	if(default_deconstruction_crowbar(user, I))
+		for(var/obj/thing in contents) // in case there is something in the scanner
```

Comment:
Oh gosh, of crap. What the heck.

You can tell this is a really *old* thing that could be constructed when you see this bullcrap.

Reply:
Yeah, this code be oooold

It's also fun removing old TODOs that never got done 😄